src/SDKs/Search/DataPlane/Microsoft.Azure.Search/generate.cmd

```diff
@@ -0,0 +1,27 @@
+::
```

Comment:
sure, we won't force this update on teams, just offer it - for some teams, the specs-repo reorg makes the old scripts not work anymore (no more composite files) so I wanted to offer a quick solution by migrating everything over. Search data-plane isn't affected by this as far as I can tell, so the old script (with the URIs adjusted to point to the reorged Swaggers) thould totally keep working.